tests/Unit/Api/Service/MyParcelApiServiceTest.php

```diff
@@ -30,7 +28,7 @@
             'Authorization' => null,
             'User-Agent'    => sprintf(
                 'MyParcel-PDK/%s php/7.4.30 Prestashop/1.7.8.6',
```

Comment:
Niet jouw pr maar waarom is er een hardcoded php en prestashop versie? Ik weet dat ik een andere Prestashop en php versie draai...